core/dragging/bubble_drag_strategy.ts

```diff
@@ -33,7 +33,9 @@ export class BubbleDragStrategy implements IDragStrategy {
     this.startLoc = this.bubble.getRelativeToSurfaceXY();
     this.workspace.setResizesEnabled(false);
     this.workspace.getLayerManager()?.moveToDragLayer(this.bubble);
-    this.bubble.setDragging && this.bubble.setDragging(true);
+    if (this.bubble.setDragging) {
+      this.bubble.setDragging(true);
+    }
```

Comment:
Could use optional chaining here:

```suggestion
    this.bubble.setDragging?.(true);
```

Reply:
Acknowledged